frontend/src/components/edition/editionRowComponent.tsx

```diff
@@ -60,29 +62,33 @@ export function EditionRowComponent(props: EditionProps) {
                     >
                         <Image alt="" src={"/resources/view.svg"} width="15" height="15" />
                     </a>
-                    <a
-                        style={{ padding: "1rem", cursor: "pointer" }}
-                        href={transformer(
-                            applicationPaths.editionBase +
-                                "/" +
-                                extractIdFromEditionUrl(edition._links.self.href)
-                        )}
-                        data-testid="list-edit-edition"
-                    >
-                        <Image
-                            alt={capitalize(t("edit"))}
-                            src={"/resources/edit.svg"}
-                            width="15"
-                            height="15"
-                        />
-                    </a>
-                    <a
-                        style={{ cursor: "pointer" }}
-                        onClick={deleteEdition}
-                        data-testid="list-delete-edition"
-                    >
-                        <Image alt="" src={"/resources/delete.svg"} width="15" height="15" />
-                    </a>
+                    {currentUserIsAdmin && (
+                        <a
+                            style={{ padding: "1rem", cursor: "pointer" }}
+                            href={transformer(
+                                applicationPaths.editionBase +
+                                    "/" +
+                                    extractIdFromEditionUrl(edition._links.self.href)
```

Comment:
Is this different form just the self.href?

Reply:
There is no edition.self.href? If that's what you mean?

Reply:
Oh, this is a frontend path, my bad!